tasks/bundle_analysis_notify.py

```diff
@@ -99,15 +109,16 @@ def process_impl_within_lock(
             # every processor errored, nothing to notify on
             notify = False
 
-        success = None
         if notify:
             installation_name_to_use = get_installation_name_for_owner_for_task(
                 db_session, self.name, commit.repository.owner
             )
-            notifier = BundleNotifier(
+            notifier = BundleAnalysisNotifyService(
                 commit, commit_yaml, gh_app_installation_name=installation_name_to_use
             )
-            success = async_to_sync(notifier.notify)()
+            result = notifier.notify()
+            configured_notifications_count = len(result.notifications_configured)
+            successful_notifications_count = len(result.notifications_successful)
```

Comment:
How about instead of getting these two lengths first and then then calling `get_success_value` on them,
you have either a `NotifyResult.to_success_value`, or alternatively a `success_value_from_result(Result)`.
That would encapsulate these things better.